customfiles/depenguin_bsdinstall.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+set -e
+# shellcheck disable=SC3040
+set -o pipefail
+
+exit_error() {
+    echo "$*" 1>&2
+    exit 1;
+}
+
+# read in variables
+if [ -f depenguin_settings.sh ]; then
+	# shellcheck source=/dev/null
+	. depenguin_settings.sh
+else
+	exit_error "Copy depenguin_settings.sh.sample to depenguin_settings.sh, edit to your needs, then run depenguin_bsdinstall.sh again"
+fi
+
+# check if template installerconfig exists
+if [ ! -f INSTALLERCONFIG.sample ]; then
+	exit_error "Missing INSTALLERCONFIG.sample. Please check location."
+fi
+
+# shellcheck disable=SC3003
+# safe(r) separator for sed
+sep=$'\001'
+
+# change variables in INSTALLERCONFIG to our settings and save to INSTALLERCONFIG.active
+# shellcheck disable=SC2154
```

Comment:
You can remove the spellcheck exclusion, once you changed /dev/null above.

this has the advantage of checking your sample file as (if the sample misses a variable, it will show up as unassigned here)